### PR TITLE
Windows installer - don't assume drive letter for `pip.exe`

### LINKED ISSUE #74: Pip dependencies have failed to install
**Branch**: 0.14 (installing on Windows 7)

**Current Behavior**: Error while installing 0.14 over 0.13 (fully functional with several coaches & learners & language packs):

Pip dependencies have failed to install.

pip-dependencies

(The spelling error has been fixed in https://github.com/learningequality/installers/pull/73)

The installation continued after this error, but is probably why the 0.14 server was unable to start at the end. I had no issues with Pip this morning when I did a clean new install on another Win 7 machine.  

**Steps to reproduce**:

1. Download & install the 0.14 on the machine with previous 0.13 installation of KA Lite

### LINKED ISSUE #81: UX Issue: "Setup will now install pip dependencies"
I clicked out before I could take a screenshot, but this was a popup that appeared after installing. Thought this line might be confusing for people who don't understand development. Could this line be changed to something easier to understand, like "configuring your installation" or something similar?

### PR DESCRIPTION
**Pull request by MCGallaspy**
Handles the case where `pip.exe` is not on the `C:` drive. Fixes #74 and fixes #81. Will test out on Windows 8. @radinamatic I'll upload an installer for you to try, as well.

### CODE REVIEW

**Comment by MCGallaspy** [comment]
Depends on #73.

**Comment by MCGallaspy** [comment]
Ugh, no, this doesn't seem to work.

**Comment by MCGallaspy** [comment]
Here's my solution:
* The installer searches for pip in `XX`:\Python27\Scripts\pip.exe, where `XX` stands for the letters "C" through "H".
* If it can't find pip in those locations, it prompts the user to select pip.exe.
* If the user cancels the prompt, the installation is aborted.
* If pip is found using either of the above methods, then installation proceeds normally.

**Comment by radinamatic** [comment]
I just now realized that there are 3 **pips** in my `D:\Python27\Scripts` folder, all with exactly the same `Date modified` attribute:
- pip.exe
- pip2.exe
- pip2.7.exe

So, who's the winner?

**Comment by MCGallaspy** [comment]
They all are.

**Comment by radinamatic** [comment]
Meaning, if the user is prompted, they can choose any of those three indifferently?

**Comment by radinamatic** [comment]
Sorry, 32bit... :(

runtime64

**Comment by MCGallaspy** [comment]
> Meaning, if the user is prompted, they can choose any of those three indifferently?

Indeed so. It makes more sense on *nix systems, where they are generally symlinks to the versioned name.

**Comment by MCGallaspy** [comment]
Regarding your other error, I'm on it.

File: windows/installer-source/KaliteSetupScript.iss [hunk 1]
```diff
@@ -417,6 +417,38 @@ begin
...
+    DEFAULT_PATH = '\Python27\Scripts\pip.exe';
+
+{ Returns the path of pip.exe on the system. }
+{ Tries several different locations before prompting user. }
+function GetPipPath: string;
+var
+    path : string;
+    i : integer;
+begin
+    for i := Ord('C') to Ord('H') do
```
Comment (aronasorman) [inline windows/installer-source/KaliteSetupScript.iss]: Why not until 'Z:\'? I believe Z: is the default drive letter for wine.

Comment (MCGallaspy) [inline windows/installer-source/KaliteSetupScript.iss]: Who is running this in wine?

Comment (MCGallaspy) [inline windows/installer-source/KaliteSetupScript.iss]: But at any rate such a user will be prompted to select the location of pip.exe later on.

**Pull request merged**